### ISSUE TITLE
Please support wxPython 2.8

### ISSUE THREAD

**Issue by Lenbok** [dkobozev/tatlin#20]
I'm no python guru, but is it possible to support wxPython 2.8, which is the version that comes standard on Ubuntu 14.04 (LTS)?

**Comment by Lenbok** [comment]
Hi, I just tried master now there is the wxPython 2.8 compatibility, but after the UI shows there is no view of the code/stl displayed. On the console are errors like this:

Traceback (most recent call last):
  File "/home2/len/dev/tatlin/libtatlin/ui.py", line 806, in _on_paint
    self.init()
  File "/home2/len/dev/tatlin/libtatlin/scene.py", line 114, in init
    self.init_actors()
  File "/home2/len/dev/tatlin/libtatlin/scene.py", line 121, in init_actors
    actor.init()
  File "/home2/len/dev/tatlin/libtatlin/actors.py", line 83, in init
    self.display_list = compile_display_list(self.draw)
  File "/home2/len/dev/tatlin/libtatlin/actors.py", line 37, in compile_display_list
    func(*options)
  File "/home2/len/dev/tatlin/libtatlin/actors.py", line 101, in draw
    for i in range(0, self.width + 1):
TypeError: range() integer end argument expected, got float.

My ~/.tatlin has this in it
```
[machine]
platform_w = 200
platform_d = 200

[ui]
window_w = 1024
window_h = 768
gcode_2d = 0
```

If I move that out of the way, a view appears (and no exceptions on the console), however the view is very small, and doesn't respond when mouse-drag around inside of it. If I resize the window, that seems to trigger a repaint showing a semi-updated view that includes GFX of the widgets. Overall, it's completely broken :-(

![image](https://cloud.githubusercontent.com/assets/282098/6613423/843e2248-c8f1-11e4-9782-a8d2b9390257.png)

**Comment by Lenbok** [comment]
(The last commit I can run is 7ac3903)

**Comment by dkobozev** [comment]
I fixed the first issue that had to do with a custom platform size specified in your .tatlin.

However, I'm a bit stumped by the rendering issue you're having. I tried running the latest commit on Ubuntu 14.04 inside a VM. I got a bunch of OpenGL warnings in the console and a completely black screen on file load, which I worked around by switching to fullscreen mode. However, these issues seem to be caused by the VM and don't match the symptoms you're describing. I was able to navigate using the mouse and resize the Tatlin window without any problems.
